test/framework/registryMirror.go

```diff
@@ -48,6 +54,35 @@ func WithRegistryMirrorEndpointAndCert() ClusterE2ETestOpt {
 	}
 }
 
+func WithTinkerbellRegistryMirrorEndpointAndCert() ClusterE2ETestOpt {
```

Comment:
Rather than creating new function can we not just use existing function and pass the appropriate env var name as params to the function?